rfhub2/db/model/keyword.py

```diff
@@ -24,3 +25,36 @@ def __repr__(self):  # pragma: no cover
     @classmethod
     def default_ordering(cls) -> List[OrderingItem]:
         return [OrderingItem("name")]
+
+    @staticmethod
+    def create(data: model.KeywordCreate) -> "Keyword":
+        return Keyword(
+            name=data.name,
+            doc=data.doc,
+            args=data.args,
+            collection_id=data.collection_id,
+        )
+
+    def to_model(self) -> model.Keyword:
+        collection = self.collection
+        return model.Keyword(
+            id=self.id,
+            name=self.name,
+            doc=self.doc,
+            args=self.args,
```

Comment:
@MaciejWiczk you could add here conversion of `tags: str` in DB model to `tags: List[str]` in `mode.Keyword` by using json deserialization